comps/cores/mega/all_manifests.yaml

```diff
@@ -0,0 +1,688 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+data:
+  EMBEDDING_MODEL_ID: BAAI/bge-base-en-v1.5
+  EMBEDDING_SERVICE_HOST_IP: embedding-svc
+  HUGGINGFACEHUB_API_TOKEN: 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
```

Comment:
Should we hide all of the `HUGGINGFACEHUB_API_TOKEN`?

Reply:
Yes. I will delete this file and revoke this HF TOKEN.